cmd/attestation-container/attestation-container.go

```diff
@@ -20,12 +20,16 @@ import (
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
+
+	"github.com/sirupsen/logrus"
 )
 
 var (
 	socketAddress             = flag.String("socket-address", "/tmp/attestation-container.sock", "The socket address of Unix domain socket (UDS)")
 	platformCertificateServer = flag.String("platform-certificate-server", "", "Server to fetch platform certificate. If set, certificates contained in security context directory are ignored. Value is either 'Azure' or 'AMD'")
 	insecureVirtual           = flag.Bool("insecure-virtual", false, "If set, dummy attestation is returned (INSECURE: do not use in production)")
+	logLevel                  = flag.String("loglevel", "warning", "Logging Level: trace, debug, info, warning, error, fatal, panic.")
+	logFile                   = flag.String("logfile", "", "Logging Target: An optional file name/path. Omit for console output.")
```

Comment:
Where does "console output" end up? Is that on the ACI log tab in the portal? If so then are you certain that nothing sensitive can ever be logged?

Reply:
We also have methods available via the security policy to outright block stdio from containers.  A customer that is worried about sensitive items in the log can enable this block.

Reply:
That being said, we should strive to make sure that sensitive items are not logged via error or warn.  I would say that debug logs would be ok to have sensitive information so long as a customer has to opt into debug.